pkgs/applications/audio/rofi-mpd/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, python37Packages, fetchFromGitHub }:
+
+with python37Packages; 
+buildPythonPackage rec {
```

if this is just meant to be an executable and not imported by other python modules, you should prefer buildPythonApplication
```suggestion
buildPythonApplication rec {
```